### PR TITLE
v0.10.0 Changelog and Release Notes

### PR DESCRIPTION
**Pull request by zhill**
Adds CHANGELOG.md update and docs updates for release notes for v0.10.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,64 @@
...
+### Breaking Changes
+*Note: the policy engine feed sync configuration is now in the policy engine service configuration as part of the provider configuration. The provided helm charts, docker-compose.yaml and default configurations handle this change automatically.*
+
+### Changes
+
++ Added - Policy engine should leverage Grype. Fixes #706 and #707
++ Added - Improved alpine vulnerability scanning by using NVD matches for OS packages for CVEs that are not yet present in Alpine SecDB. Fixes #268
++ Fixed - Adds missing fields and fixes errors in the swagger spec for the API
++ Fixed - Restores file package verification data ingress during image load to fix a regression. Fixes #965
++ Fixed - Malware policy gate can fail causing policy eval error when malware not enable and other rules precede malware rule in a policy. Fixes #992
```
Comment (dspalmer99) [inline CHANGELOG.md:52]: * not enabled

File: docs/content/docs/releasenotes/0100.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+### Breaking Changes
+*Note: the policy engine feed sync configuration is now in the policy engine service configuration as part of the provider configuration. The provided helm charts, docker-compose.yaml and default configurations handle this change automatically.*
+
+### Changes
+ 
++ Added - Policy engine should leverage Grype. Fixes #706 and #707
++ Added - Improved alpine vulnerability scanning by using NVD matches for OS packages for CVEs that are not yet present in Alpine SecDB. Fixes #268
++ Fixed - Adds missing fields and fixes errors in the swagger spec for the API
++ Fixed - Restores file package verification data ingress during image load to fix a regression. Fixes #965
++ Fixed - Malware policy gate can fail causing policy eval error when malware not enable and other rules precede malware rule in a policy. Fixes #992
```
Comment (dspalmer99) [inline docs/content/docs/releasenotes/0100.md:53]: Same as above, not enable -> not enabled

**Review by dspalmer99 — APPROVED** [review]
One minor typo in two places - but it is in the title of an issue that was closed. Otherwise LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,64 @@
 # Changelog
 
+## 0.10.0
+
+API version - 0.1.18
+
+DB Schema version - 0.0.15
+
+This release contains a database schema update, but no data migration only new tables added.
```
Comment (dspalmer99) [inline CHANGELOG.md:9]: Nit: The second clause in this sentence would read a little better with a conjunction in it, something like "but no data migration as only new tables were added.

**Comment by zhill** [comment]
Realized I'm missing the configurable filtering by package ownership, will add that.

**Review by Vijay-P — APPROVED** [review]
LGTM

**Pull request merged**